changes/changes-8.2.0.md

```diff
@@ -87,6 +87,7 @@
 * Fix end time bug in the query process.
 * Fix `Exporter INCREMENT mode` is not working.
 * Fix an error when executing startup.bat when the log directory exists
+* Add syncBulkActions config to avoid the large amount of metrics data written ES in a single sync bulk request.
```

Comment:
I think this description is not correct, you just separate the config, can't avoid that thing

Reply:
Could change this to 

> Add syncBulkActions configuration to set up the batch size of the trace/log persistent.

Reply:
Done.